Concentration/ViewController.swift

```diff
@@ -89,7 +96,9 @@ class ViewController: UIViewController {
                       "Flags": (["🏳️‍🌈", "🏳️", "🏴","🏴‍☠️","🏁","🇧🇿","🇫🇯","🇹🇭","🇮🇳"], #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1), #colorLiteral(red: 0.2588235438, green: 0.7568627596, blue: 0.9686274529, alpha: 1))
                     ]
     
-    lazy var emojiChoices = themesDict.randomElement()!
+    // The theme of the game (default theme is halloween).
+    lazy var emojiChoices = themesDict.randomElement() ??
```

Comment:
This is a bad practice. If "Halloween" theme will be change in the future, you’ll have to update it in both places.
You can avoid the hole logic here and just pick a random theme when starting a new game.